.azure/gpu-pipeline.yml

```diff
@@ -70,11 +78,13 @@ jobs:
         sudo apt-get install -y build-essential gcc g++ cmake ffmpeg git libsndfile1 unzip --no-install-recommends
         # python -m pip install "pip==20.1"
         pip install -e . --requirement ./requirements/devel.txt
+        pip install pandas --force-reinstall
         pip install mkl-service==2.4.0  # needed for the gpu multiprocessing
         pip list
       displayName: 'Install environment'
 
     - bash: |
+        python -c "from torch import __version__ as ver ; assert str(ver) == '$(torch-ver)', f'PyTorch: {ver}'"
```

Comment:
@SkafteNicki turned out what we were always assumed to be testing LTS (1.8.1) was, in fact always reinstalled with the latest/stable :see_no_evil: